{{cookiecutter.project_slug}}/clients/web/vue3/src/services/AxiosClient.js

```diff
@@ -18,7 +18,7 @@ class ApiService {
   static session
   static init
   constructor() {
-    let baseURL = `${window.location.protocol}//${window.location.host}/api`
+    let baseURL = `${window.location.protocol}//${window.location.host}/`
 
```

Comment:
Thoughts on this? We do not append the `api` to the domain in the react side, I think that is a better approach since it is more explicit and controlled by the dev

Reply:
My thought would be the opposite. The AxiosClient is making all of it's calls to the backend API.
The baseURL for the backend API starts at `/api`.

Reply:
I'm with Ed on this one, I'd prefer /api stayed there

Reply:
sounds good will update appropriately